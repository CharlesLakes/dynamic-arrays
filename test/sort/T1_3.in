10
72 97 74 62 31 46 5 59 73 8