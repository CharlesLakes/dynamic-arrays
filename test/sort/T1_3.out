5 8 31 46 59 62 72 73 74 97